.buildkite/scripts/integration_tests.sh

```diff
@@ -7,14 +7,8 @@ set -euo pipefail
 
 ensure_logout() {
     local error_code=$?
-
-    if [ $error_code != 0 ] ; then
-        # if variable is defined run the logout
-        if [ -n "${GOOGLE_APPLICATION_CREDENTIALS+x}" ]; then
-             google_cloud_logout_active_account
-        fi
-    fi
-
+    echo "GCP logout is not required, the BK plugin will do it for us"
+    echo "Error code: ${error_code}"
     exit $error_code
 }
 trap ensure_logout EXIT
```

Comment:
It looks like that this function could be removed as it only adds a message. WDYT ?

Reply:
See [`0b98b51` (#2569)](https://github.com/elastic/elastic-package/pull/2569/commits/0b98b51af35ddad5715415a942c6fa13af8070a8)

Reply:
See [`c1ecdc7` (#2569)](https://github.com/elastic/elastic-package/pull/2569/commits/c1ecdc7e1987c2dc15996ffe57511c2480be0f2c)